docs/requirements.txt

```diff
@@ -3,3 +3,4 @@ furo
 Sphinx>4
 sphinx-copybutton
 sphinxext-opengraph
+requests>=2.2
```

Comment:
Don't think this is needed in here as these are requirements specific to building the docs.